qemu/tests/sr_iov_hotplug.py

```diff
@@ -113,24 +116,52 @@ def device_add(pci_num, pci_add_cmd):
                       ". Output: %r" % (pci_add_cmd, add_output))
         return after_add
 
+    def clean_network_scripts():
+        logging.debug("Clean up network scripts in guest")
+        session = vm.wait_for_serial_login(timeout=timeout)
```

Comment:
many sessions are not closed.

Reply:
@CongLi as suggested, I have closed the session.

Reply:
I have closed the session in `finally`